SingularityRunnerBase/src/main/java/com/hubspot/singularity/runner/base/config/MissingConfigException.java

```diff
@@ -0,0 +1,7 @@
+package com.hubspot.singularity.runner.base.config;
+
+public class MissingConfigException extends IllegalStateException {
```

Comment:
Yeah that works, I had done IllegalState to start with because thats what Preconditions was throwing before. I'll update to Runtime